docs/architecture/overview.md

```diff
@@ -24,16 +24,18 @@ Aggregators have knowledge of the different data source interfaces in the data t
 
 For example, the `Carbon Intensity Aggregator` handles requests for various carbon emissions information. It can calculate the average carbon emissions over a time period, or the best carbon emissions given a set of locations. It can transform forecasted carbon emissions to suit particular use-cases. It can also just deliver the emissions data points in a standard schema without performing any calculations.
```

Comment:
Another missed reference to the `Carbon Intensity Aggregator`. This whole paragraph might need a quick rewriting